src/sonic-yang-models/tests/yang_model_tests/tests_config/acl.json

```diff
@@ -1003,5 +1003,41 @@
                 ]
             }
         }
+    },
+    "ACL_RULE_VALID_TCP_FLAGS": {
+        "sonic-acl:sonic-acl": {
+            "sonic-acl:ACL_RULE": {
+                "ACL_RULE_LIST": [
+                    {
+                        "ACL_TABLE_NAME": "TCP_FLAGS_TEST",
+                        "ETHER_TYPE": "2048",
+                        "PACKET_ACTION": "DROP",
+                        "PRIORITY": 9981,
+                        "TCP_FLAGS": "0x24",
+                        "RULE_NAME": "Rule_19"
+                    },
+                    {
+                        "ACL_TABLE_NAME": "TCP_FLAGS_TEST",
+                        "ETHER_TYPE": "2048",
+                        "PACKET_ACTION": "DROP",
+                        "PRIORITY": 9981,
+                        "TCP_FLAGS": "0x24/0x24",
+                        "RULE_NAME": "Rule_20"
+                    }
+
+                ]
+            },
+            "sonic-acl:ACL_TABLE": {
+                "ACL_TABLE_LIST": [
+                    {
+                        "ACL_TABLE_NAME": "TCP_FLAGS_TEST",
+                        "policy_desc": "TCP_FLAGS_TEST",
+                        "ports": [ "" ],
+                        "stage": "INGRESS",
```

Comment:
What's the purpose of this test case?

Reply:
RULE test need to be corellated with a TABLE.